src/prog_models/models/test_models/linear_models.py

```diff
@@ -67,6 +67,16 @@ class OneInputOneOutputNoEventLM(LinearModel):
     }
 
 
+class OneInputOneOutputNoEventLMPM(OneInputOneOutputNoEventLM):
+    """
+    Same as OneInputOneOutputNoEventLM, but with performance metrics defined as a function. Has a single performance metric that is always the state, plus 1
+    """
+    performance_metric_keys = ['x1+1']
```

Comment:
I find this notation to be slightly confusing/hard to read when we implement it in the tests below (e.g. with syntax like 'OneInputOneOutputNoEventLMPM.x1+1'). What if we changed it to something like, 'x1plus1' ? Open to suggestions